docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,197 @@
+
+## Submitting issues
+
+All issues related to the associated Ansible modules, roles, playbooks, regardless of the service/repository the issue belongs to (see table above), should be submitted [here](https://github.com/dell/<<repo>>/issues). Issues will be triaged and labels will be used to indicate the type of issue. This section outlines the types of issues that can be submitted.  
+
+### Report bugs
+
+We aim to track and document everything related to the repository via the Issues page. The code and documentation are released with no warranties or SLAs and are intended to be supported through a community driven process.
+
+Before submitting a new issue, make sure someone hasn't already reported the problem. Look through the [existing issues](https://github.com/dell/<<repo>>/issues) for similar issues.
+
+Report a bug by submitting a [bug report](https://github.com/dell/<<repo>>/issues/new?labels=type%2Fbug%2C+needs-triage&template=bug_report.md&title=%5BBUG%5D%3A). Make sure that you provide as much information as possible on how to reproduce the bug.
+
+When opening a Bug please include the following information to help with debugging:
+
+1. Version of relevant software: this software, Python version, Dell Server/Storage Platform, etc.
+2. Details of the issue explaining the problem: what, when, where
+3. The expected outcome that was not met (if any)
+4. Supporting troubleshooting information. __Note: Do not provide private company information that could compromise your company's security.__
+
+An Issue __must__ be created before submitting any pull request. Any pull request that is created should be linked to an Issue.
+
+### Feature request
+
+If you have an idea of how to improve this project, submit a [feature request](https://github.com/dell/<<repo>>/issues/new?labels=type%2Ffeature-request%2C+needs-triage&template=feature_request.md&title=%5BFEATURE%5D%3A).
+
+### Answering questions
+
+If you have a question and you can't find the answer in the documentation or issues, the next step is to submit a [question.](https://github.com/dell/<<repo>>/issues/new?labels=type%2Fquestion&template=ask-a-question.md&title=%5BQUESTION%5D%3A)
+
+We'd love your help in answering pending questions asked by other CSM users.
+
+## Triage issues
+
+Triage helps ensure that issues resolve quickly by:
+
+* Ensuring the issue's intent and purpose is conveyed precisely. This is necessary because it can be difficult for an issue to explain how an end user experiences a problem and what actions they took.
+* Giving a contributor the information they need before they commit to resolving an issue.
+* Lowering the issue count by preventing duplicate issues.
```

Comment:
```suggestion

```

Reply:
removed as suggested

Reply:
This is another change that apparently was made and was subsequently reversed. I am unresolving this issue.

Please figure out how this happened to prevent future similar recurrences.

To resolve this issue, either remove this line or modify it as suggested in this issue.

Reply:
addressed as suggested